public/dorequest.php

```diff
@@ -306,7 +308,12 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
                 $activityMessage = utf8_sanitize($activityMessage);
             }
 
-            PlayerSessionHeartbeat::dispatch($user, $game, $activityMessage);
+            $gameHashMd5 = request()->post('x') ?? request()->query('x') ?? null;
```

Comment:
Isn't this just:
```
$gameHashMd5 = request()->input('x');
```

Reply:
Yep. I managed to forget that `request()->input()` was a thing. Updated in latest.